README.md

```diff
@@ -112,37 +130,46 @@ RUN chgrp -R 0 /var/run/foo
 * When building an image, the building context is the `docker/images/<service>`
   directory, so if you need to add files to the context (_e.g._ for a `COPY`
   statement), make sure to place them in this directory.
+* Don't forget to add and update the `LABEL` statement as proposed above to
+  fully qualify your container.
 
 ### Publish your image using the CI
 
 Once your image is ready to be published, you are invited to:
 
-1. Update the list of available images in the next section of this document.
+1. Update the list of available images in the next section of this document
+   (don't forget to add [microbadger](https://microbadger.com) badges).
 2. Push your feature-branch (you've created a feature branch, right?) to GitHub
    and open a new pull request (PR).
 3. Look for CI status and wait for a review of your work. If everything went
    well, proceed to the next step.
 4. Create a new repository on DockerHub under the `fundocker` organization
    umbrella (it should be named following our image tagging pattern - see
    above), and give the `bot` team `write` access to this repository.
-5. Tag the repository (see building strategy in the CI/CD section) to publish
-   your image:
+5. Merge your PR.
```

Comment:
Create a Pull Request before merging ?

Reply:
I don't understand this comment, the PR has been opened in step 2.

Reply:
Oh It's a mistake :-(